src/hatch_protobuf/plugin.py

```diff
@@ -50,6 +50,9 @@ def initialize(self, version: str, build_data: Dict[str, Any]) -> None:
         for generator in self._generators:
             args.append(f"--{generator.name}_out={generator.output_path}")
 
+        for include in self._includes:
+            args.append(f"-I{include}")
```

Comment:
i'd rather use `--proto_path` like with `self._proto_paths` above (and also either merge it with that loop or put it immediately next to that loop) - `-I` and `--proto_path` do the same thing, but i'd like to be consistent about which we use